### PR TITLE
Update technology.py

### LINKED ISSUE #108: Rewrite mortality method
Mort values in specs are per 100,000 people, need to be changed per person. Additionally, check morb.

Change the pafs of the function to use the rural and urban ones. Check #107

### PR DESCRIPTION
**Pull request by babakkhavari**
This fixes #108

### CODE REVIEW

File: onsstove/technology.py [hunk 1]
```diff
@@ -150,59 +148,38 @@ def mortality(self, specs_file, paf_0_alri, paf_0_copd, paf_0_lc, paf_0_ihd):
...
-        mort_lc_U = specs_file["Urban_HHsize"] * (paf_0_lc - paf_lc) * specs_file["Mort_LC"]
-
-        mort_alri_R = specs_file["Rural_HHsize"] * (paf_0_alri - paf_alri) * specs_file["Mort_ALRI"]
-        mort_copd_R = specs_file["Rural_HHsize"] * (paf_0_copd - paf_copd) * specs_file["Mort_COPD"]
-        mort_ihd_R = specs_file["Rural_HHsize"] * (paf_0_ihd - paf_ihd) * specs_file["Mort_IHD"]
-        mort_lc_R = specs_file["Rural_HHsize"] * (paf_0_lc - paf_lc) * specs_file["Mort_LC"]
+        mort_alri = specs_file["Population_start_year"] * (paf_0_alri - paf_alri) * (specs_file["Mort_ALRI"]/100000)
+        mort_copd = specs_file["Population_start_year"] * (paf_0_copd - paf_copd) * (specs_file["Mort_COPD"]/100000)
+        mort_ihd = specs_file["Population_start_year"] * (paf_0_ihd - paf_ihd) * (specs_file["Mort_IHD"]/100000)
+        mort_lc = specs_file["Population_start_year"] * (paf_0_lc - paf_lc) * (specs_file["Mort_LC"]/100000)
```
Comment (camiloramirezgo) [inline onsstove/technology.py:154]: If we pass the main `gdf` as an input and change `specs_file["Population_start_year"]` for `gdf['Calibrated_pop']` we would get the mort and morb for each cell right?

File: onsstove/technology.py [hunk 2]
```diff
@@ -150,59 +148,38 @@ def mortality(self, specs_file, paf_0_alri, paf_0_copd, paf_0_lc, paf_0_ihd):
...
 
         i = 1
-        mort_U_vector = []
-        mort_R_vector = []
+        mort_vector = []
         while i < 6:
-            mortality_alri_U = cl_alri[i] * specs_file["VSL"] * mort_alri_U / (1 + specs_file["Discount_rate"]) ** (i - 1)
-            mortality_copd_U = cl_copd[i] * specs_file["VSL"] * mort_copd_U / (
+            mortality_alri = cl_alri[i] * specs_file["VSL"] * mort_alri_U / (1 + specs_file["Discount_rate"]) ** (i - 1)
+            mortality_copd = cl_copd[i] * specs_file["VSL"] * mort_copd_U / (
```
Comment (camiloramirezgo) [inline onsstove/technology.py:165]: I think here we need to remove U from `mort_alri_U`.

File: onsstove/technology.py (hunk 1 above)
Comment (babakkhavari) [reply onsstove/technology.py:154]: yeah that is true, you want us to change to that instead? both here and in morb?

File: onsstove/technology.py (hunk 2 above)
Comment (babakkhavari) [reply onsstove/technology.py:165]: yes

File: onsstove/technology.py (hunk 1 above)
Comment (babakkhavari) [reply onsstove/technology.py:154]: And then I assume we need to sum it for the equations below that

Comment (camiloramirezgo) [reply onsstove/technology.py:154]: Yes I think so

**Pull request merged**